appengine/standard-analytics/Gemfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "https://rubygems.org"
+
+gem "sinatra"
```

Comment:
Please add a semver-oriented version restriction to sinatra for this and all other Gemfiles. e.g.

```
gem "sinatra", "~> 2.1"
```

And please regenerate the corresponding Gemfile.lock files.

Reply:
done